frontend/packages/dev-console/integration-tests/support/pages/helm/controls.ts

```diff
@@ -0,0 +1,18 @@
+export const controls = {
+  dropdown: {
+    switchTo: (newOption: string) => {
+      cy.byLegacyTestID('dropdown-button')
+        .click()
+        .get('.pf-c-dropdown__menu')
+        .contains(newOption)
+        .click();
+    },
+    verifyVisibility: () => {
+      cy.byLegacyTestID('dropdown-button').should('be.visible');
+      cy.byLegacyTestID('dropdown-button')
+        .click()
+        .get('.pf-c-dropdown__menu')
+        .should('be.visible');
+    },
+  },
+};
```

Comment:
Can you please rename the function name to suite functionality

Reply:
as this is handled in command, I would to like to remove this file. https://github.com/openshift/console/pull/7795#discussion_r567772922